definition/wof_controlled.md

```diff
@@ -0,0 +1,28 @@
+## What is the wof:controlled property used for?
+
+Many Who’s On First records have a `wof:controlled` property, a top-level property used to "freeze" other property values in a given Who’s On First record. The `wof:controlled` property is a list that can contain any other existing property keys in a Who’s On First record. The most common use of this property is to "freeze" the `wof:hierarchy` and `wof:parent_id` properties, as follows:
```

Comment:
Many Who’s On First records have a `wof:controlled` property that is used to maintain curated property values in a given Who’s On First record during automatic updates and other editing tasks. Extreme caution should be exercised when changing values in properties that are marked controlled.